### PR TITLE
Tiled Gallery Block: Update styles for WP.com emails.

### PR DESCRIPTION
**Pull request by kwight**
Adding these styles will improve rendering of the Tiled Gallery block in WordPress.com subscriber emails.

Fixes 196-gh-Automattic/dotcom-manage

Before | After
--- | ---
Screen Shot 2020-11-03 at 1 55 50 PM | Screen Shot 2020-11-03 at 1 55 33 PM

#### Changes proposed in this Pull Request:

*

#### Jetpack product discussion
None.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Apply this patch to your WP.com sandbox using the Fusion-generated diff.
* On a WordPress.com test site, publish a post with a Jetpack Tiled Gallery block.
* Send a test email for this post (see PCYsg-tvN-p2).
* Verify the results look good. It doesn't have to be an exact layout match, the images should just make some visual sense and not be crowded.

#### Proposed changelog entry for your changes:
No changelog needed.

### CODE REVIEW

**Review by blackjackkent — DISMISSED** [review]
Looking good!

image

**Comment by krymson24** [comment]
Looks good!
Screen Shot 2020-11-08 at 5 53 42 PM

**Review by krymson24 — APPROVED** [review]

**Comment by kwight** [comment]
Anyone from @Automattic/jetpack-approvers able to look at this please?

**Review by fgiannar — APPROVED** [review]
LGTM 👍

**Review by jeherve — APPROVED** [review]

**Review by jeherve — APPROVED** [review]
Approving one last time after a final rebase to fix unrelated test failures.

**Comment by blackjackkent** [comment]
@jeherve Thank you for jumping on this! Was about to come around with questions about that wpcom status, haha. Much appreciated.

**Comment by jeherve** [comment]
r216886-wpcom

**Pull request merged**
